src/Ledger/Certs.lagda

```diff
@@ -92,25 +88,17 @@ cwitness (reg c (suc _))     = just c
 \begin{AgdaMultiCode}
 \begin{code}
 record CertEnv : Type where
-\end{code}
-\begin{code}[hide]
   constructor ⟦_,_,_,_,_⟧ᶜ
```

Comment:
Constructors will now be visible.  I think that's safer in case we decide to leave some occurrences of them unadulterated (i.e., not converted to a vertical vector).

Reply:
Also, once @carlostome completes his type class for these constructors, they will disappear anyway.